### PR TITLE
implement dcrlibwallet get tx, improve same function for wallet rpc 

### PR DESCRIPTION
**Pull request by itswisdomagain**
The output of cli `show-transaction` via the updated rpc function or the newly implemented dcrlibwallet function. Major changes are addition of account name when displaying outputs and displaying block height instead of block hash. The block height is gotten using the block hash and since the blockhash is invalid (see #96), the block height is not currently found. A default value of -1 is displayed whenever the block hash cannot be used to get a matching block height.

screenshot 2019-01-05 at 8 17 19 am

UPDATE:
dcrwallet rpc block hash issue is resolved, and block heights for transactions now display

### CODE REVIEW

**Review by ademuanthony — CHANGES_REQUESTED** [review]
When the transaction hash is not found, the error in the output is too verbose, can we catch the error code from the response just print transaction not found?
not-found
I suspect the error to be of `dcrdwallet/errors.Error`

File: app/walletmediums/dcrwalletrpc/walletfunctions.go [hunk 1]
```diff
@@ -324,18 +354,29 @@ func (c *WalletPRCClient) TransactionHistory() ([]*walletcore.Transaction, error
...
-	getTxResponse, err := c.walletService.GetTransaction(context.Background(), getTxRequest)
+	getTxResponse, err := c.walletService.GetTransaction(ctx, getTxRequest)
 	if err != nil {
 		return nil, err
 	}
 
-	transactionHex := fmt.Sprintf("%x", getTxResponse.GetTransaction().GetTransaction())
-	msgTx, err := txhelpers.MsgTxFromHex(transactionHex)
+	// func to attempt to check if an address belongs to the wallet to retrieve it's account name or return external if not
+	getWalletAddressInfo := func(address string) *txhelper.AddressInfo {
```
Comment (barisere) [inline app/walletmediums/dcrwalletrpc/walletfunctions.go]: This function looks like an exact copy of [this one here](https://github.com/raedahgroup/godcr/pull/127/files#diff-b081aa982693a57870847580a9b9cd71R266). The can be extracted into a package-level function.

**Review by barisere — CHANGES_REQUESTED** [review]
#96 has been fixed, see https://github.com/raedahgroup/godcr/issues/96#issuecomment-451650393

**Comment by itswisdomagain** [comment]
Block height now display with the latest versions of dcrd and dcrwallet.
screenshot 2019-01-05 at 9 42 28 pm

Transaction not found errors now show simpler error message:
`transaction not found`

File: app/walletmediums/dcrwalletrpc/walletfunctions.go [hunk 2]
```diff
@@ -379,6 +382,7 @@ func (c *WalletPRCClient) GetTransaction(transactionHash string) (*walletcore.Tr
 
 	var blockHeight int32 = -1
 	if getTxResponse.BlockHash != nil {
+		//fmt.Printf("%x\n", getTxResponse.GetBlockHash())
```
Comment (barisere) [inline app/walletmediums/dcrwalletrpc/walletfunctions.go]: The comment is not needed any longer.

**Review by barisere — CHANGES_REQUESTED** [review]

**Review by ademuanthony — APPROVED** [review]

**Review by barisere — APPROVED** [review]

**Pull request merged**